bin/site-inspector

```diff
@@ -1,15 +1,29 @@
 #!/usr/bin/env ruby
 
 require_relative "../lib/site-inspector"
+
 require "oj"
 
 domain = ARGV[0]
+http_mode = (ARGV[1] == "--http")
```

Comment:
For small scripts like this, `ARGV` is the best route IMHO. There's nothing great our there, short of something like Mercenary which is a bit heavy weight.